### PR TITLE
Enhancement UI

### PR DESCRIPTION
**Pull request by PhyrexTsai**
## UI enhancement
- Footer style 
- Search Bar UI and style

### CODE REVIEW

File: src/components/Footer/Footer.css [hunk 1]
```diff
@@ -1,3 +1,11 @@
-
+.Footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-end;
+  height: 64px;
+  background-color: #4A148C !important;
+  position: absolute !important;
+  bottom: 0; }
```
[the remark this answers is not in the document]
Comment (chochinlu) [reply src/components/Footer/Footer.css]: Sorry I should write this comment in the SCSS file.  :P

File: src/components/SearchEns/SearchEns.scss [hunk 2]
```diff
@@ -2,12 +2,36 @@
...
-  font-size: 1.8em;
-  padding: 5px 1em;
-  border: 1px solid black;
+  font-size: 1.2em;
+  padding-left: 1em;
+  padding-right: 1em !important;
+  height: 2em;
+  width: 100%;
+  width: -moz-available;
+  width: -webkit-fill-available;
```
Comment (chochinlu) [inline src/components/SearchEns/SearchEns.scss]: the `create-react-app`'s webpack will help us to take care the cross browser issues.

File: src/components/SearchEns/SearchEns.scss [hunk 3]
```diff
@@ -2,12 +2,36 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 70%;
+  min-width: 380px;
+}
+
+.SearchEns-typography {
+  font-size: 1.2em !important;
+  color: rgba(0, 0, 0, 0.54) !important;
```
Comment (chochinlu) [inline src/components/SearchEns/SearchEns.scss]: (Suggestion) Should we uniform the css color style?  use rgba or hex value?

File: src/components/SearchEns/SearchEns.scss (hunk 2 above)
Comment (PhyrexTsai) [reply src/components/SearchEns/SearchEns.scss]: I tried, but I have to config this then its works.

File: src/components/SearchEns/SearchEns.scss (hunk 3 above)
Comment (PhyrexTsai) [reply src/components/SearchEns/SearchEns.scss]: Yes, I just try using my own style to config UI now, we will move to color css

File: src/components/SearchEns/SearchEns.scss (hunk 2 above)
Comment (chochinlu) [reply src/components/SearchEns/SearchEns.scss]: I see. merge soon. :P

**Pull request merged**